diff --git a/src/internal/unix/bash.js b/src/internal/unix/bash.js
index 19fba6d..c1c57ce 100644
--- a/src/internal/unix/bash.js
+++ b/src/internal/unix/bash.js
@@ -15,7 +15,7 @@ function escapeArg(arg) {
     .replace(/\n/gu, " ")
     .replace(/\\/gu, "\\\\")
     .replace(/(?<=^|\s)([#~])/gu, "\\$1")
-    .replace(/(["$&'()*;<>?`{|])/gu, "\\$1")
+    .replace(/(["$&'()*;<>?`{|[\]])/gu, "\\$1")
     .replace(/(?<=[:=])(~)(?=[\s+\-/0:=]|$)/gu, "\\$1")
     .replace(/([\t ])/gu, "\\$1");
 }
diff --git a/src/internal/unix/busybox.js b/src/internal/unix/busybox.js
index e61262f..0c91dc4 100644
--- a/src/internal/unix/busybox.js
+++ b/src/internal/unix/busybox.js
@@ -15,7 +15,7 @@ function escapeArg(arg) {
     .replace(/\n/gu, " ")
     .replace(/\\/gu, "\\\\")
     .replace(/(?<=^|\s)([#~])/gu, "\\$1")
-    .replace(/(["$&'()*;<>?`|])/gu, "\\$1")
+    .replace(/(["$&'()*;<>?`|[\]])/gu, "\\$1")
     .replace(/([\t ])/gu, "\\$1");
 }
 
